torch_frame/data/multi_embedding_tensor.py

```diff
@@ -0,0 +1,166 @@
+import copy
+from typing import Any, Callable, Sequence, Tuple
+
+import torch
+from torch import Tensor
+
+
+class MultiEmbeddingTensor:
+    r"""A PyTorch tensor-based data structure that stores
+    :obj:`[num_rows, num_cols, *]`, where the size of last dimension can be
+    different for different column.
+
+    Args:
+        num_rows (int): Number of rows.
+        num_cols (int): Number of columns.
+        values (torch.Tensor): The values :class:`torch.Tensor` of size
+            :obj:`[num_rows, dim1+dim2+...+dimN]`.
+        offset (torch.Tensor): The offset :class:`torch.Tensor` of size
+            :obj:`[num_cols+1,]`.
+    """
+    def __init__(
+        self,
+        num_rows: int,
+        num_cols: int,
+        values: Tensor,
+        offset: Tensor,
+    ) -> None:
```

Comment:
Don't we need a `classmethod` here to construct `MultiEmbeddingTensor` much more easily, e.g.:
```python
MultiEmbeddingTensor.from_list(embeddings: List[Tensor])` where `len(embeddings) == num_cols`.

Reply:
+1. I think we can follow the pattern of `MultiNestedTensor`

Reply:
DOne!